foundry/pixi/containers/ruler.d.ts

```diff
@@ -1,167 +1,184 @@
-declare interface RulerData {
-  _state: number;
-  class: string;
-  destination: PIXI.Point;
-  name: string;
-  waypoints: [PIXI.Point];
-}
-
 /**
  * The Ruler - used to measure distances and trigger movements
- * @param user - The User for whom to construct the Ruler instance
+ * @param user  - The User for whom to construct the Ruler instance
+ * @param color - The color for the ruler
+ *                (Default value: The user's color or `0x42f4e2`)
  */
 declare class Ruler extends PIXI.Container {
+  constructor(user?: User, { color }?: { color?: number | null });
+
   /**
-   * Track the current measurement state
+   * Record the User which this Ruler references
    */
-  public _state: number;
+  user: User;
 
   /**
-   * Is the Ruler being actively used to measure distance?
+   * The ruler name - used to differentiate between players
    */
-  public readonly active: boolean;
+  name: string;
 
   /**
    * The ruler color - by default the color of the active user
    */
-  public color: number | null;
+  color: number | null;
```

Comment:
This is documented as possibly `null` but I don't think it actually can be `null` there is a fallback value:
```js
    this.color = color || colorStringToHex(this.user.data.color) || 0x42F4E2;
```